ebpf/redirect.bpf.c

```diff
@@ -49,7 +49,7 @@ check_skip_process_map_entry(uint32_t pid)
 inline __attribute__((always_inline)) int
 update_audit_map_entry(bpf_sock_addr_t *ctx)
 {
-    uint64_t pid_tip = bpf_sock_addr_get_current_pid_tgid(ctx);
+    uint64_t pid_tip = bpf_get_current_pid_tgid(ctx);
```

Comment:
we dont need to pass ctx here. Sample code: https://github.com/microsoft/ebpf-for-windows/blob/e5ebe0d1ab73dc0a697081c454f8436b12883432/tests/sample/cgroup_sock_addr2.c#L44